### ISSUE TITLE
Multiple Checkbox with options

### ISSUE THREAD

**Issue by silici0** [thedevdojo/voyager#4483]
## Version information

 - Laravel: v6.0.1
 - Voyager: v1.2.7
 - PHP: 7.2.8
 - Database: MySQL 5.6.43

## Description:
When use Input Type Multiple Checkbox on BREAD, with options, after save a few options, on Edit it come everything selected ( If you save and didn't check the options all options became selected ) 

## Screenshots:
Options on BREAD :
![Captura de Tela 2019-10-16 às 17 37 50](https://user-images.githubusercontent.com/61994/66956980-c62bb280-f03b-11e9-84e3-138097f76cef.png)

After add a product with two options selected it saved just fine :
![Captura de Tela 2019-10-16 às 17 39 49](https://user-images.githubusercontent.com/61994/66957158-1276f280-f03c-11e9-8270-386131ef38b6.png)

When i edit the same product the problem (every option is selected) : 
![Captura de Tela 2019-10-16 às 17 41 07](https://user-images.githubusercontent.com/61994/66957218-2a4e7680-f03c-11e9-9ea1-da51a22f0d14.png)

**Comment by silici0** [comment]
The problem is on resources/views/formfields/multiple_checkbox.blade.php

On line 6 
`<?php $checked = old($row->field, $dataTypeContent->{$row->field}); ?>`

To fix 
`<?php $checked = (in_array($key, (array)json_decode($dataTypeContent->{$row->field}))); ?>`

**Comment by MrCrayon** [comment]
That way you loose data if there is a validation error, it should be rather something like this:
```blade
            @php
                $checkedData = old($row->field, $dataTypeContent->{$row->field});
                $checkedData = is_array($checkedData) ? $checkedData : json_decode($checkedData, true);
                $checked = in_array($key, $checkedData);
            @endphp
```

**Comment by emptynick** [comment]
Closing as https://github.com/the-control-group/voyager/pull/4484 was merged (into 1.3)
